.bcr/metadata.template.json

```diff
@@ -0,0 +1,14 @@
+{
+    "homepage": "https://github.com/grpc/grpc",
+    "maintainers": [
+        {
+            "email": "bcr-maintainers@bazel.build",
```

Comment:
Can we add proper contacts here? See protobuf as an example: https://github.com/protocolbuffers/protobuf/blob/main/.bcr/metadata.template.json